### ISSUE TITLE
UI lag and random freeze

### ISSUE THREAD

**Issue by gandriyko** [kivymd/KivyMD#77]
Hello!
I use latest kivy, KivyMD and tried to build application [kitchen_sink](https://github.com/HeaTTheatR/KivyMD/tree/master/demos/kitchen_sink), but this application UI looks slow. All touch actions occur with notable delay, even clicking on buttons
I install [KIvyMD Demo](https://play.google.com/store/apps/details?id=org.kivymd.kivymd&hl=en) from Google Play and it works fine. 
I also tried to build the [kivy showcase](https://github.com/kivy/kivy/tree/master/examples/demo/showcase) with same buildozer.spec and it works perfectly.
Could you suggest me what is wrong?
I use latest kivy/buildozer on Ubuntu 18.04
Attached video and buildozer.spec file
[kivy-data.zip](https://github.com/HeaTTheatR/KivyMD/files/3415039/kivy-data.zip)

**Comment by HeaTTheatR** [comment]
@gandriyko All examples in the navigation box are loaded dynamically when you select one of the examples. Download occurs once. All subsequent times already loaded examples open quickly. If I downloaded the KV file with examples completely in the program start process, the program would open for ten years. The work of the interface elements itself works quite normally and I did not notice any brakes.
